api-implementation/apiScreens/RecepiDetailScreen.js

```diff
@@ -37,14 +41,110 @@ function RecepiDetailScreen() {
             setLoading(true); // Reset loading state when screen is focused
             fetchData();
         }
-    }, [isFocused,navigation])
-
+    }, [isFocused, navigation])
     if (isloading) return <ActivityIndicator style={{ alignItems: "center" }} size="large" color="#e4baa1" />
```

Comment:
create a separate constant files for storing constant values (eg: Colors, value)

Reply:
Added Colors file